### PR TITLE
Api800/fc network

### PR DESCRIPTION
**Pull request by soodpr**
### Description
API 800 Support for FC Networks

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for go 1.11 + gofmt checks .
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [X] New endpoints supported are updated in the endpoints-support.md file.
- [] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/fc_networks.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		apiversion)
+	initialScopeUris := new([]utils.Nstring)
+	*initialScopeUris = append(*initialScopeUris, utils.NewNstring("/rest/scopes/8a4e85fe-0725-482e-b232-4877b78fde18"))
```
Comment (patrickdappollonio) [inline examples/fc_networks.go]: You can change these two lines to:

```go
initialScopeUris :=[]utils.Nstring{utils.NewNstring("/rest/scopes/8a4e85fe-0725-482e-b232-4877b78fde18")}
```

File: examples/fc_networks.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	}
+	sort := "name:desc"
+
+	fcNetworks, err := ovc.GetFCNetworks("", sort, "", "")
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("#---Get Fc Networks sorted by name in descending order----#")
+
+	for i := 0; i < len(fcNetworks.Members); i++ {
```
Comment (patrickdappollonio) [inline examples/fc_networks.go]: Consider using a `for ... range` loop, since it won't require you to access items by index.

Comment (soodpr) [reply examples/fc_networks.go]: done. Thanks

File: examples/fc_networks.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	} else {
+		fmt.Println("Fc Network created successfully...")
+	}
+	sort := "name:desc"
+
+	fcNetworks, err := ovc.GetFCNetworks("", sort, "", "")
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("#---Get Fc Networks sorted by name in descending order----#")
```
Comment (madhav-bharadwaj) [inline examples/fc_networks.go]: As discussed yesterday, please put this inside a else.

File: examples/fc_networks.go [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+	for i := range fcNetworks.Members {
+		fmt.Println(fcNetworks.Members[i].Name)
+	}
+
+	fcNetwork2, err := ovc.GetFCNetworkByName(testName)
+	if err != nil {
+		fmt.Println(err)
+	}
+	fmt.Println("#-------------Get FCNetworks by name----------------#")
```
Comment (madhav-bharadwaj) [inline examples/fc_networks.go]: same as above

**Pull request merged**